src/js/utils/stylesheet.js

```diff
@@ -0,0 +1,16 @@
+import document from 'global/document';
+
+export let getStyleElement = function(className) {
```

Comment:
This should probably be named `createStyleElement`, otherwise it looks like you're getting an existing style el with the supplied class name.